src/main/java/seedu/address/model/Model.java

```diff
@@ -171,4 +171,9 @@
      * @param room The room from which the task should be delete.
      */
     void deleteTaskFromRoom(Task task, Room room);
+
```

Comment:
Maybe consider "returns" instead of "gets" for javadoc consistency?

Reply:
Okay!